tracer.go

```diff
@@ -92,7 +92,15 @@ func parseTagsAsZipkinOptions(t map[string]interface{}) []zipkin.SpanOption {
 		default:
 			kind = fmt.Sprintf("%v", kindVal)
 		}
-		zopts = append(zopts, zipkin.Kind(model.Kind(strings.ToUpper(kind))))
+		mKind := model.Kind(strings.ToUpper(kind))
```

Comment:
This is it @adriancole

Reply:
tx

Reply:
do we want this?.. if model.Kind does not have a banana option it simply becomes "" (empty)

by adding this logic we need to sync whenever we add a new kind option in zipkin-go